packages/devtools_app/lib/src/service/resolved_uri_manager.dart

```diff
@@ -0,0 +1,51 @@
+import '../shared/globals.dart';
+import 'vm_service_wrapper.dart';
+
+/// Manager for handling package Uri lookup and caching.
+class ResolvedUriManager {
+  Map<String, String?>? _resolvedUrlMap;
```

Comment:
question for @bkonyi - should we be storing a cache per isolate? right now this would store package uris for all isolates in the same cache

Reply:
It's _probably_ fine (I doubt URI mappings for files will be different between isolates in practice), but it's more correct to have a cache per isolate since that's how things are setup in the VM.

Reply:
ty, I have made it so!